luthier/config/src/lib/json.lua

```diff
@@ -398,4 +398,4 @@ function json.decode(str)
 end
 
 
-return json
+return json
```

Comment:
Looks like you removed a trailing newline at the end of this file. That should probably be put back.